models/settings_helpers.go

```diff
@@ -152,6 +154,8 @@ func UpdateSettings(q reform.DBTX, params *ChangeSettingsParams) (*Settings, err
 		settings.SaaS.Email = params.Email
 	}
 
+	settings.VictoriaMetrics.CacheEnable = params.VMCacheEnable
```

Comment:
Yep, we want to disable it by default.

Reply:
No, my point is that ChangeSettings API can be called with just `{}` (empty JSON object) – it is a supported way to re-read base files, regenerate configuration, etc. With that change, that will always disable cache. Again, that why we have DisableSTT / EnableSTT as two separate booleans instead of just one.

Reply:
Thanks, get it. Changed logic - added DisableVMCache param.